lib/generator-author.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+var nunjucks = require('nunjucks');
```

Comment:
I assume you use nunjucks because something else does. The project isn't currently maintained.